locales/en.js

```diff
@@ -66,6 +66,9 @@ export default {
   privacyLink: "https://www.canada.ca/en/transparency/privacy.html",
   privacyText: "Privacy",
 
```

Comment:
The link for prime minister currently redirects to http://localhost:3000/pm.gc.ca/en. Works as intended if you change the link to `http://pm.gc.ca/en`

Same thing with the french link